requirements.txt

```diff
@@ -1,5 +1,5 @@
 Django~=2.2.0
-celery>=4.0.2,<=4.4.0,!=4.1.0
```

Comment:
Do you know why 4.1.0 was excluded? I'd imagine that we ran into an issue with it at some point.

Reply:
Added it back 👍